apps/frontend/src/components/global/upload_tabs/SampleList.vue

```diff
@@ -39,20 +39,25 @@ export default class SampleList extends Vue {
       align: 'start',
       sortable: true,
       value: 'filename'
+    },
+    {
+      text: 'Type',
+      value: 'fingerprint',
+      sortable: true
     }
   ];
 
   loading = false;
 
-  load_samples(sampleArray: Sample[]) {
+  load_samples(samples: Sample[]) {
     this.loading = true;
     Promise.all(
-      sampleArray.map((arrayItem) => {
-        return arrayItem.data().then((data: JSON) => {
-          return InspecIntakeModule.loadText({
-            text: JSON.stringify(data),
-            filename: arrayItem.filename
-          });
+      samples.map<Promise<FileID>>((sample: Sample) => {
```

Comment:
these types aren't correct, we do need to account for the fact that loadfile returns promise<fileid | fileid[]>

Reply:
I believe I may have fixed this by including a .flat in the .then. Also updated the code for readbility.